src/Pelagos/Bundle/AppBundle/Resources/public/js/dif.js

```diff
@@ -606,59 +607,99 @@ function updateDIF(form)
+            } else if (statusCode == 1) {
+                var title = "DIF Submitted";
+                var message = '<div><img src="' + imgInfo + '">' +
+                    "<p>Congratulations! You have successfully submitted a DIF to GRIIDC. The UDI for this dataset is " + udi + "." +
+                    "<br>The DIF will now be reviewed by GRIIDC staff and is locked to prevent editing. To make changes" +
+                    "<br>to your DIF, please email GRIIDC at griidc@gomri.org with the UDI for your dataset." +
+                    "<br>Please note that you will receive an email notification when your DIF is approved.</p></div>";
+            } else if (status.statusCode == 2) {
+                var title = "DIF Updated and Approved";
+                var message = '<div><img src="' + imgInfo + '">' +
+                    "<p>The application with DIF ID: " + udi + " was successfully updated and approved!" +
+                    "<br></p></div>";
+            }
+        } else if (status.message === "error") {
+            if (status.statusCode == 3) {
+                var title = "Unable to approve DIF";
+                var message = '<div><img src="' + imgInfo + '">' +
+                    "<p>The application with DIF ID: " + udi + " cannot be approved as it is already approved!" +
+                    "<br></p></div>";
+            } else {
+                var title = "Unable to perform desired action on DIF";
+                var message = '<div><img src="' + imgInfo + '">' +
+                    "<p>The application with DIF ID: " + udi + " failed to complete action!" +
+                    "<br></p></div>";
+            }
         }
 
         hideSpinner();
         formReset(true);
         //loadDIFS();
 
-        $("<div>"+message+"</div>").dialog({
-            autoOpen: true,
-            resizable: false,
-            minWidth: 300,
-            height: "auto",
-            width: "auto",
-            modal: true,
-            title: title,
-            buttons: {
-                OK: function() {
-                    $(this).dialog("close");
-                    scrollToTop();
-                    treeFilter();
-                    return $.Deferred().resolve();
+        if (status.message === "error") {
+            $("<div>" + message + "</div>").dialog({
+                autoOpen: true,
+                resizable: false,
+                minWidth: 300,
+                height: "auto",
+                width: "auto",
+                modal: true,
+                title: title,
+                buttons: {
+                    OK: function () {
+                        $(this).dialog("close");
+                        scrollToTop();
+                        treeFilter();
+                        return $.Deferred().resolve();
+                    }
                 }
-            }
-        });
+            }).parent().addClass("ui-state-error");
```

Comment:
Instead of using this class (which is reserved for jQuery validate, just change the imgInfo in the message to another icon.

Reply:
That way you can also just use one of these dialog sections.

Reply:
Removed this